### PR TITLE
(#618) Update AU docs and links

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

- Update links and docs mentioning AU.

## Motivation and Context

As AU is no longer being maintained by the original owner and we have forked it with the intent that it be maintained by the chocolatey community

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

#618

### CODE REVIEW

File: input/en-us/create/automatic-packages.md [hunk 1]
```diff
@@ -16,16 +16,19 @@ Automatic packaging is a process that **package maintainers can run on *their ow
 
 There are currently two methods that can be used to maintain automatic packages:
 
-- RECOMMENDED - Using only Powershell via [Automatic Update module AU](https://github.com/majkinetor/au).
+- RECOMMENDED - Using only Powershell via [Automatic Update module AU](https://github.com/chocolatey-community/chocolatey-au).
```
Comment (pauby) [inline input/en-us/create/automatic-packages.md:19]:
```suggestion
- RECOMMENDED - Using only Powershell via [Chocolatey Automatic Package Updater Module](https://github.com/chocolatey-community/chocolatey-au).
```

Comment (pauby) [reply input/en-us/create/automatic-packages.md:19]: This is consistent with the text below.

**Review by pauby — CHANGES_REQUESTED** [review]

**Comment by pauby** [comment]
LGTM! Can you rebase @vexx32 ?

**Comment by vexx32** [comment]
Sorted 🙂

File: input/en-us/create/automatic-packages.md [hunk 2]
```diff
@@ -16,16 +16,19 @@ Automatic packaging is a process that **package maintainers can run on *their ow
 
 There are currently two methods that can be used to maintain automatic packages:
 
-- RECOMMENDED - Using only Powershell via [Automatic Update module AU](https://github.com/majkinetor/au).
+- RECOMMENDED - Using only Powershell via [Chocolatey Automatic Package Updater Module](https://github.com/chocolatey-community/chocolatey-au).
```
Comment (gep13) [inline input/en-us/create/automatic-packages.md:19]:
```suggestion
- RECOMMENDED - Using only PowerShell via [Chocolatey Automatic Package Updater Module](https://github.com/chocolatey-community/chocolatey-au).
```

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a small suggestion...

File: input/en-us/create/automatic-packages.md (hunk 2 above)
Comment (vexx32) [reply input/en-us/create/automatic-packages.md]: Amended, thanks!

**Review by pauby — APPROVED** [review]

**Pull request merged**